src/patternfly/components/Accordion/accordion-expanded-content-body.hbs

```diff
@@ -0,0 +1,6 @@
+<p class="pf-c-accordion__expanded-content-body{{#if accordion-expanded-content-body--modifier}} {{accordion-expanded-content-body--modifier}}{{/if}}"
```

Comment:
Should this be a `<p>`? The content model for a paragraph only allows [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content). So they couldn't use a `<div>`, `<h1>`, `<ul>`, etc. I think this would be better as a `<div>`.